### DISCUSSION TITLE
Comments on the speaker overview paper

### DISCUSSION THREAD

**Discussion by wsstriving** [wenet-e2e/wespeaker#339] [General]
Dear all,
We are initiating a discussion on the paper "[Overview of Speaker Modeling and Its Applications: From the Lens of Deep Speaker Representation Learning](https://arxiv.org/pdf/2407.15188)" here. Due to the page limitation (26 pages), there might be some topics that are not covered sufficiently in this paper.
**We hope to draw inspiration from the valuable discussion, comments, and suggestions to improve the arXiv version** (which has no strict page limitation). Especially, we are looking forward to receiving comments from industry professionals to uncover the real-world challenges and applications related to speaker modeling that are yet covered.
The comments can cover, but are not limited to, the following aspects:

1. Important topics that are not covered.
2. Important references that are missing.
3. The challenges that we didn't mention but are truly significant in real-world applications.
4. Possible questionable or outdated expressions.
5. Trends of this area in your opinion.
6. More.

Many thanks in advance!

**Comment by wsstriving** [comment]
update dataset info about voxblink2

**Comment by wsstriving** [comment]
update about the latest improvement of open-source toolkits

**Comment by Snowdar** [comment]
感谢分享SV领域的综述。只说知道的部分，提一个工具描述和表格部分的更正：asv-subtools 支持c++部署，同时kaldi只是特征提取和训练的可选工具，核心net部分一开始就是pytorch，也支持online dataloader使用python提取特征，因此也是容易做自定义的。

**Reply by wsstriving** [reply]
Will update the description the next version
